### ISSUE TITLE
Export server: possible memory leak?

### ISSUE THREAD

**Issue by lpsnc** [highcharts/highcharts#3786]
After stopping a (long running) export server process we get the following `SEVERE` message about a likely memory leak. Should we be concerned about that? 

```
Feb 03, 2015 2:42:44 PM org.apache.coyote.http11.Http11Protocol pause
INFO: Pausing Coyote HTTP/1.1 on http-11000
Feb 03, 2015 2:42:45 PM org.apache.catalina.core.StandardService stop
INFO: Stopping service Catalina
Feb 03, 2015 2:42:45 PM org.apache.catalina.loader.WebappClassLoader checkThreadLocalMapForLeaks
SEVERE: The web application [/highcharts-export-web] created a ThreadLocal with key of type [com.google.gson.Gson$1] (value [com.google.gson.Gson$1@219116eb]) and a value of type [java.util.HashMap] (value [{}]) but failed to remove it when the web application was stopped. This is very likely to create a memory leak.
Feb 03, 2015 2:42:45 PM org.apache.coyote.http11.Http11Protocol destroy
INFO: Stopping Coyote HTTP/1.1 on http-11000
```

**Comment by gvaartjes** [comment]
Probably resolved by upgrading to 2.2.4 
read last comment in this thread; https://code.google.com/p/google-gson/issues/detail?id=402

**Comment by KacperMadej** [comment]
The problem seems to be about legacy export server - please use Node export server and in case of found bug in that export server please report on the correct repository - https://github.com/highcharts/node-export-server

If you are able to recreate this issue, then please report this on the legacy export server repository - https://github.com/highcharts/highcharts-export-server
